### PR TITLE
bootc: Implement the 'install' subcommand

### PR DESCRIPTION
**Pull request by dcantrell**
Implement the 'install' subcommand for the bootc plugin and add documentation.  Most of the options for this subcommand will come from global dnf command line option settings.  The only option not handled is --os=OSNAME from rpm-ostree.

https://issues.redhat.com/browse/SWM-3090

### CODE REVIEW

File: plugins/bootc.py
Comment (evan-goode) [inline plugins/bootc.py:77]: rpm-ostree accepts multiple `--uninstall` args, looks like

```rpm-ostree install foo --uninstall bar --uninstall baz```

so this should be `nargs="+"`

**Review by evan-goode — CHANGES_REQUESTED** [review]

File: doc/bootc.rst
Comment (evan-goode) [inline doc/bootc.rst:41]: should also clarify this supports multiple pkgspecs if we want to support that, see below comment

**Comment by evan-goode** [comment]
Looks like we're missing the PKGSPEC args:

```
# dnf4 bootc install hello
usage: dnf bootc [-c [config file]] [-q] [-v] [--version] [--installroot [path]] [--nodocs] [--noplugins] [--enableplugin [plugin]] [--disableplugin [plugin]] [--releasever RELEASEVER] [--setopt SETOPTS] [--skip-broken] [-h] [--allowerasing] [-b | --nobest] [-C] [-R [minutes]] [-d [debug level]] [--debugsolver]
                 [--showduplicates] [-e ERRORLEVEL] [--obsoletes] [--rpmverbosity [debug level name]] [-y] [--assumeno] [--enablerepo [repo]] [--disablerepo [repo] | --repo [repo]] [--enable | --disable] [-x [package]] [--disableexcludes [repo]] [--repofrompath [repo,path]] [--noautoremove] [--nogpgcheck]
                 [--color COLOR] [--refresh] [-4] [-6] [--destdir DESTDIR] [--downloadonly] [--comment COMMENT] [--bugfix] [--enhancement] [--newpackage] [--security] [--advisory ADVISORY] [--bz BUGZILLA] [--cve CVES] [--sec-severity {Critical,Important,Moderate,Low}] [--forcearch ARCH] [--json] [--booted]
                 [--jsonpath EXPRESSION] [--pending-exit-77] [--uninstall PKG] [-A] [--force-replacefiles] [-r] [--allow-inactive] [--idempotent] [--unchanged-exit-77] [--peer]
                 [BOOTC]
dnf bootc: error: unrecognized arguments: hello
```

File: doc/bootc.rst
Comment (dcantrell) [reply doc/bootc.rst:20]: Ah, good call.

File: plugins/bootc.py
Comment (dcantrell) [reply plugins/bootc.py:77]: Also good catch.

File: doc/bootc.rst
Comment (dcantrell) [reply doc/bootc.rst:41]: Might as well.

**Comment by dcantrell** [comment]
Addressed these findings and pushed an updated version of the patch.

File: doc/bootc.rst
Comment (evan-goode) [inline doc/bootc.rst:39]: just noticed "deploymewnt", might as well fix it now

File: plugins/bootc.py
Comment (evan-goode) [inline plugins/bootc.py:201]: This breaks `bootc status`, this check should only be on commands that expect pkgspecs:

```
# dnf4 bootc status
Missing package specification on bootc sub-command status.
```

**Review by evan-goode — CHANGES_REQUESTED** [review]

**Comment by evan-goode** [comment]
Nice, should be good to go after merge conflicts are resolved and `bootc status` is fixed. This subcommand will be more useful after https://github.com/rpm-software-management/dnf-plugins-core/pull/558 is merged, currently the user has to wait a few minutes before getting any output.

File: doc/bootc.rst
Comment (dcantrell) [reply doc/bootc.rst:39]: OK, if you insiwst,

File: plugins/bootc.py
Comment (dcantrell) [reply plugins/bootc.py:201]: Whoops.

**Comment by dcantrell** [comment]
Thanks for the second review.  I think I have everything taken care of now.

**Review by evan-goode — APPROVED** [review]

**Pull request merged**
